### PR TITLE
Add Overlap stub

### PR DESCRIPTION
**Pull request by Pennek**
Still need input on how tags should look

### CODE REVIEW

File: wiki/Mapping_Techniques/Overlap/en.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Overlap
+
+![Rotational_Symmetry.png](/img/overlap.jpg "An example of a hitcircle overlapping a sliderhead.")
```
Comment (MegaApplePi) [inline wiki/Mapping_Techniques/Overlap/en.md]:
```suggestion
![Rotational_Symmetry.png](img/overlap.jpg "An example of a hitcircle overlapping a sliderhead.")
```

the beginning `/` in the link basically means to start with the root folder, to use current directory remove it or use `./`

**Comment by MegaApplePi** [comment]
Temporarily blocking for now. I want to see the general game plan and other specifics in #3143 first before jumping the gun on this.


You also mentioned about something needing input on how tags should look. Do you mean that `stub: true` part?

**Comment by Pennek** [comment]
no problem for the blocking. these take little effort.

yes the stub. I want that cleared up before pushing these forward, since we dont have a conses on how to add them for now

File: wiki/Mapping_Techniques/Overlap/en.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Overlap
+
+![overlap.png](./img/overlap.jpg "An example of a hitcircle overlapping a sliderhead.")
```
Comment (MegaApplePi) [inline wiki/Mapping_Techniques/Overlap/en.md]: The alt text should be a description of the image. If you don't have one, it should be left blank.

Comment (Pennek) [reply wiki/Mapping_Techniques/Overlap/en.md]: what more would you add to this? the curvature of the slider? it simply is a hitcircle overlapping a sliderhead

Comment (MegaApplePi) [reply wiki/Mapping_Techniques/Overlap/en.md]: Just to be sure we are talking about the same thing,
```
![overlap.png](./img/overlap.jpg "An example of a hitcircle overlapping a sliderhead.")
  ^ This is the alt text          ^ This is the title/hover text, also used as the caption on web
```

Alt text only displays when the image doesn't load, but it is also used for accessibility purposes too. If you don't have an image description for the alt text, then don't put anything in there.

Comment (Pennek) [reply wiki/Mapping_Techniques/Overlap/en.md]: that clears it up. so leave it blank?

Comment (MegaApplePi) [reply wiki/Mapping_Techniques/Overlap/en.md]: Yeah, that'll be fine.

**Comment by cl8n** [comment]
for now it's probably best to do what Loctav suggested, add a note at the bottom saying this is a stub. I don't think the "stub: true" thing is ever going to turn into a web feature cuz this is starting to get pretty redundant with all of these `marker: true` tags that have similar effects

**Comment by cl8n** [comment]
@MegaApplePi this would be 1. from https://github.com/ppy/osu-wiki/issues/3143#issuecomment-581197907 , and even if that were never to be completed i dont think adding these would hurt anything

all that comes to mind for me is that we may need to revisit how indices and Sitemap display this kind of stuff.

**Comment by MegaApplePi** [comment]
Here are some notes on what was discussed through discord regarding the stub mark:

image

---

For now, we'll mark stubs with an html comment to help identify them in the future. If you think a stub can be further expanded, you could also include the note that clayton wrote at the bottom.

**Pull request merged**